interface/weno/admin.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+
+$sanitize_all_escapes = true;		// SANITIZE ALL ESCAPES
+
+$fake_register_globals = false;		// STOP FAKE REGISTER GLOBALS
+
+require_once('../globals.php');
+require_once('transmitDataClass.php');
+require_once("adminClass.php");
+
+$isActive = new transmitData();
+$tables   = new adminProperties();
+
+$active = $isActive->active();
+$exist  = $tables->dataBaseTableExist();
+
+//var_dump($exist);
+
+?>
+<html>
+<head>
+     <title><?php print xlt("Weno Admin"); ?></title>
+    <?php html_header_show(); ?>
+    <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+
+</head>
+
+
+<body class="body_top">
+<?php 
+
+if($active['gl_value'] != 1){ print xlt("You must activate Weno first!"); exit; } else {print xlt("Weno Service is Enabled")."<br>";} 
```

Comment:
Isn't this just checking a global setting? If so, should just check the setting directly.